### ISSUE TITLE
Loss fluctuation during pretraining

### ISSUE THREAD

**Issue by sinatayebati** [Nightmare-n/GD-MAE#21]
After pretraining the model on KITTI and plotting the loss, I noticed lots of fluctuations between ~min of 0.1 to ~max of 0.8, it appears that the loss does not stabilize nor saturate. Is this a usual trend and have you experienced the same training situation?

**Comment by Nightmare-n** [comment]
Hi, I have trained the model on KITTI-360 before. From what I remember, the losses seem to converge normally.
